readme.txt

```diff
@@ -37,7 +36,12 @@ We're always interested in your feedback and our [Help Desk](https://support.the
 
 == Changelog ==
 
+= [1.0.2] TBD =
+
+* Actual working code.
+
 = [1.0.1] 2020-04-09 =
+
 * Test version change.
```

Comment:
eww ;)

Can we delete the remote tag? Run this on the repo in your Terminal and then re-release actual 1.0.1 in the changelog with accurate release date.

`git push --delete origin 1.0.1`

Reply:
Actually, all the `since` tags are v1.0.0 so just make sure github releases/tags are accurate along with changelog

Reply:
Yep - was using it to test the GitHub updater plugin for Courtney's article. (she needed screenshots)

Reply:
I deleted it. I will create a new release once we go live with this.